scripts/build/debug/300-gdb.sh

```diff
@@ -113,6 +113,7 @@ do_debug_gdb_build() {
             --with-build-sysroot="${CT_SYSROOT_DIR}"    \
             --with-sysroot="${CT_SYSROOT_DIR}"          \
             --disable-werror                            \
+            --disable-tui								\
```

Comment:
No need to change crosstool-ng proper, just add CT_GDB_CROSS_EXTRA_CONFIG_ARRAY="--disable-tui" to the config.